audio-worklet/basic/one-pole.html

```diff
@@ -9,9 +9,21 @@
   </head>
   <body>
     <a href="../index.html" class="link-small">&#171; AudioWorklet home</a>
-    <h1>
-      Demo: "One Pole Filter"
-    </h1>
+    <section>
+      <h1>
+        Demo: "One Pole Filter"
+      </h1>
+      <dt>An <a href="https://ccrma.stanford.edu/~jos/fp/One_Pole.html">one-pole
+      </a>filter implementation with AudioWorkletNode.</dt>
```

Comment:
Explain what the source is, and what's happening to the filter.  Otherwise, you don't know what the output should sound like.